lib/Doctrine/DBAL/Connection.php

```diff
@@ -1294,19 +1298,44 @@ public function query()
      * @throws DBALException
      */
     public function executeUpdate($query, array $params = [], array $types = [])
+    {
+        return $this->executeStatement($query, $params, $types);
+    }
+
+    /**
+     * Executes an SQL statement with the given parameters and returns the number of affected rows.
+     *
+     * Could be used for:
+     *  - DML statements: INSERT, UPDATE, DELETE, etc.
+     *  - DDL statements: CREATE, DROP, ALTER, etc.
+     *  - DCL statements: GRANT, REVOKE, etc.
+     *  - Session control statements: ALTER SESSION, SET, DECLARE, etc.
+     *  - Other statements that don't yield a row set.
+     *
+     * This method supports PDO binding types as well as DBAL mapping types.
+     *
+     * @param string                 $sql    The statement SQL.
+     * @param array<mixed>           $params The query parameters.
+     * @param array<int|string|null> $types  The parameter types.
```

Comment:
Those periods at the end of non-sentences are enforced by our CS, right? I really don't get why we do this…

Reply:
This looks more like an exception, however, I believe a full stop is enforced in exception messages. Let me see if I can get rid of them here.

Reply:
I must have been confusing with some other coding standard